spec/index.html

```diff
@@ -353,14 +355,22 @@ <h2>Basic Concepts</h2>
     <h2>Core Requirements</h2>
 
     <p>
-      A YAML-LD document is a [[YAML]] document that can be interpreted as Linked Data [[LINKED-DATA]].
+      A YAML-LD stream is a YAML stream of YAML-LD documents.
+      Note that each document in a stream is independent
+      from the others:
+      each one has its own context, YAML directives,
+      named anchors and so on.
+    </p>
+    <p>
+      A YAML-LD document is a [[YAML]] document
```

Comment:
For consistency with earlier content, there should be no "YAML document", but I wonder whether that means that there should also be no YAML-LD document -- i.e., whether YAML-LD must be streams just like YAML itself.
```suggestion
      A YAML-LD document is [[YAML]]
```

Reply:
I'm leaving this suggestion unmerged based on @ioggstream's comments on today's meeting. It can be re-added as an issue or PR if it is inconsistent.